### DISCUSSION TITLE
meta tag in Head from next/head is not working in app router next js

### DISCUSSION THREAD

**Discussion by dashranjin** [vercel/next.js#68692] [Help]
### Verify canary release

- [X] I verified that the issue exists in the latest Next.js canary release

### Provide environment information

```bash
Operating System:
  Platform: win32
  Arch: x64
  Version: Windows 10 Pro
  Available memory (MB): 16090
  Available CPU cores: 16
Binaries:
  Node: 20.10.0
  npm: N/A
  Yarn: N/A
  pnpm: N/A
Relevant Packages:
  next: 14.2.4 // There is a newer version (14.2.5) available, upgrade recommended!
  eslint-config-next: 14.2.4
  react: 18.3.1
  react-dom: 18.3.1
  typescript: 5.5.3
Next.js Config:
  output: N/A
 ⚠ There is a newer version (14.2.5) available, upgrade recommended!
   Please try the latest canary version (`npm install next@canary`) to confirm the issue still exists before 
creating a new issue.
   Read more - https://nextjs.org/docs/messages/opening-an-issue
```

### Which example does this report relate to?

layout.tsx

### Describe the Bug

the meta tags in next js page router was working fine and not in app router layout

### Expected Behavior

import "@/styles/globals.css";
import type { AppProps } from "next/app";
import Head from "next/head";

export default function App({ Component, pageProps }: AppProps) {
  return (
    <>
      <Head>
        <title>Ttitle</title>
        <meta charSet="utf-8" />

        <meta
          name="viewport"
          content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no"
        />
      </Head>
      <Component {...pageProps} />
    </>
  );
}

### To Reproduce

"use client";
import { Rubik } from "next/font/google";
import "./globals.css";
import Head from "next/head";
const rubik = Rubik({
  subsets: ["latin", "cyrillic"],
  weight: ["300", "400", "500", "600", "700", "800", "900"],
});

export default function RootLayout({
  children,
}: Readonly<{
  children: React.ReactNode;
}>) {
  return (
    <>
      <html lang="en">
        <Head>
          <title>Title</title>
          <meta charSet="utf-8" />
          <meta
            name="viewport"
            content="width=device-width, initial-scale=1.0, viewport-fit=cover, user-scalable=no"
          />
        </Head>
        <body className={rubik.className + " fixed h-screen w-screen"}>
          {children}
        </body>
      </html>
    </>
  );
}

**Comment by tiberiutudorancea** [comment]


**Reply by joelrb** [reply]
This "export const metadata: Metadata" does not work if the layout is marked as "use client".

Using `<Head>` doesn't work at all. Nothing shown in the page source of all browsers!

Using latest NextJS version.

![Screenshot 2024-09-30 164607](https://github.com/user-attachments/assets/2ae457d3-3595-4ff5-99ca-ab2872939ba9)

Nothing!!!
![Screenshot 2024-09-30 163242](https://github.com/user-attachments/assets/bd0dcfa9-c973-4501-ad12-f5491dda1242)

**Comment by sujan-s** [comment]
Yeah, what gives? Why can’t I do this? Why is this a restriction?

**Comment by kiinoo** [comment]
any solutions? it is annoying
